new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -55,75 +83,142 @@ public static void collectLogs(Object itInstance, List namespaces) {
    *
    * @param namespace in which to query cluster for artifacts
    * @param resultDir existing directory to write log files
-   * @throws IOException when writing to log files fail
-   * @throws ApiException when Kubernetes cluster query fails
    */
-  public static void generateLog(String namespace, Path resultDir) throws IOException, ApiException {
+  public static void collectLogs(String namespace, String resultDir) {
     logger.info("Collecting logs in namespace : {0}", namespace);
 
     // get service accounts
-    writeToFile(Kubernetes.listServiceAccounts(namespace), resultDir.toString(), namespace + "_sa.log");
+    try {
+      writeToFile(Kubernetes.listServiceAccounts(namespace), resultDir, namespace + "_sa.log");
```

Comment:
let's improve the file names to make it easier for someone looking at them to know what they are, e.g. `namespace.list.service-accounts.log`  and so on..

namespace, get/list/whatever, type, name

Reply:
fixed